unstable/propertyvalue/ignore_changes.go

```diff
@@ -21,7 +21,7 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/resource"
 )
 
-func applyIgnoreChanges(old, new resource.PropertyMap, ignoreChanges []string) (resource.PropertyMap, error) {
+func ApplyIgnoreChanges(old, new resource.PropertyMap, ignoreChanges []string) (resource.PropertyMap, error) {
```

Comment:
Moving this around, kind of wondering are we sure there's no primitives in pulumi/pulumi that compute this or something like this? I found resource.PropertyValue has Contains now which interpreters matching against a pattern.